www/html5.appcache

```diff
@@ -1,615 +1,615 @@
-app/LogoutController.js
-app/main.js
-app/NotificationController.js
-app/OfflineController.js
-app/RestoreDatabaseController.js
-app/RoomplanController.js
-app/ScenesController.js
-app/SetupController.js
-app/TemperatureController.js
-app/TemperatureCustomLogController.js
-app/TimerplanController.js
-app/UpdateController.js
-app/UsersController.js
-app/UserVariablesController.js
-app/UtilityController.js
-app/WeatherController.js
-app/ZWaveTopologyController.js
-js/angular-animate.min.js
-js/angular-md5-min.js
-js/angular-round-progress-directive.js
-js/angular-route.min.js
-js/angular-sanitize.min.js
-js/angular-scrollglue.js
-js/angular-tree-control.js
-js/angular.min.js
-js/angularAMD.min.js
-js/blockly_compressed.js
-js/blocks_compressed.js
-js/bootbox.min.js
-js/bootstrap.min.js
-js/colpick.js
-js/d3.min.js
-js/dataTables.tableTools.min.js
-js/date.format.js
-js/domoticz.js
-js/domoticzblocks.js
-js/domoticzblocks_messages_en.js
-js/domoticzdevices.js
-js/en_compressed.js
-js/export-csv.js
-js/FileSaver.js
-js/highcharts-more.js
-js/highcharts-ng.min.js
-js/highcharts.js
-js/html5shiv.js
-js/i18next-1.8.0.min.js
-js/jquery-1.12.0.min.js
-js/jquery-ui.min.js
-js/jquery.dataTables.min.js
-js/jquery.ddslick.min.js
-js/jquery.isotope.min.js
-js/jquery.jeditable.mini.js
-js/jquery.ui.touch-punch.min.js
-js/jquery.uix.multiselect.min.js
-js/jquery-ui-timepicker-addon.js
-js/md5.js
-js/modules/exporting.js
-js/ng-grid-flexible-height.js
-js/ng-grid.min.js
-js/ngDraggable.js
-js/ngload.min.js
-js/noty/jquery.noty.js
-js/noty/layouts/top.js
-js/noty/layouts/topLeft.js
-js/noty/layouts/topRight.js
-js/noty/themes/default.js
-js/require.js
-js/respond.min.js
-js/themes/dark-blue.js
-js/themes/dark-ucust.js
-js/ui-bootstrap.min.js
-js/wow.min.js
-js/ZeroClipboard.js
-secpanel/js/less.js
-secpanel/js/ion.sound.min.js
-ozwcp/cp.js
-
-#Dynamic theme
-#ThemeFiles
-
-#Dynamic floorplans
-#Floorplans
-
-#Dynamic Switch Icons
-#SwitchIcons
-
-# HTML pages
-views/about.html
-views/cam.html
-views/customicons.html
-views/dashboard.html
-views/devices.html
-views/dpfibaro.html
-views/dphttp.html
-views/dpinflux.html
-views/dpgooglepubsub.html
-views/events.html
-views/frontpage_example.html
-views/floorplanedit.html
-views/floorplans.html
-views/hardware.html
-views/history.html
-views/lights.html
-views/log.html
-views/notification.html
-views/offline.html
-views/restoredatabase.html
-views/roomplan.html
-views/scenes.html
-views/setup.html
-views/temperature.html
-views/temperature_widget.html
-views/temperature_custom_temp_log.html
-views/timerplan.html
-views/update.html
-views/users.html
-views/utility.html
-views/weather.html
-views/weather_widget.html
-eventsframe.html
-zwavetopology.html
-secpanel/index.html
-views/login.html
-
-# Resources that require the user to be online.
-NETWORK:
-index.html
-json.htm
-ozwcp/ozwcp.html
-ozwcp/cp.html
-*
-
-# Fallback content
-FALLBACK:
-images images/iphone-icon.png
-/ views/offline.html
+CACHE MANIFEST

```

Comment:
Something as gone wrong patching the version number as every line seems to have a change, maybe it is the line ending.
Could you recheck this ?

Reply:
Line ending !